sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroSource.java

```diff
@@ -59,6 +59,7 @@
 import org.apache.commons.compress.compressors.snappy.SnappyCompressorInputStream;
 import org.apache.commons.compress.compressors.xz.XZCompressorInputStream;
 import org.apache.commons.compress.utils.CountingInputStream;
+import sun.misc.IOUtils;
```

Comment:
sun.misc.IOUtils isn't portable and may not exist on certain JDKs, find a different readFully